editoast/src/core/pathfinding.rs

```diff
@@ -133,6 +133,7 @@ pub enum PathfindingInputError {
     RollingStockNotFound {
         rolling_stock_name: String,
     },
+    ZeroLengthPath,
```

Comment:
Each pathfinding error needs a translation in `front/public/locales/$lang/errors.json`.